### PR TITLE
first two pages working

### PR DESCRIPTION
**Pull request by edumoran**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,36 +5,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Mixtape</title>
+    <title>Mixtape Setup</title>
+
+    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
```
Comment (jibm82) [inline index.html]: @edumoran looks to me that we are including the bootstrap css two times. Maybe we should remove one reference.

File: setup.html [hunk 2]
```diff
@@ -5,18 +5,31 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Setup</title>
+    <title>Mixtape Setup</title>
+
+    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
```
Comment (jibm82) [inline setup.html]: @edumoran Same here (about the duplicated bootstrap reference)

File: assets/css/css/reset.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (jibm82) [inline assets/css/css/reset.css]: @edumoran This file is not used, can we remove it?

**Review by jibm82 — CHANGES_REQUESTED** [review]
@edumoran Great work, I just left you a couple of comments for you to resolve before approval.

File: index.html [hunk 4]
```diff
@@ -5,36 +5,46 @@
...
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
```
Comment (jibm82) [inline index.html]: There is a reference to jquery at the bottom of the body, maybe we should remove this one.

File: index.html [hunk 5]
```diff
@@ -5,36 +5,46 @@
...
-                "&redirect_uri=" + redirect_uri +
-                "&scope=" + scopes.join(" ") +
-                "&response_type=token" +
-                "&show_dialog=true";
-
-            window.location = url;
-        });
-    </script>
+
+    <header class="masthead d-flex">
```
Comment (jibm82) [inline index.html:17]: Both header and the next div are missing the closing tags.

File: index.html [hunk 6]
```diff
@@ -5,36 +5,46 @@
...
+        <div class="container text-center my-auto">
+            <img src="assets/images/logo_no-shadow.png" class="img-fluid mx-auto d-block" id="mixtape">
+            <a class="btn btn" id="authorize" href="#">Start the party</a>
```
Comment (jibm82) [inline index.html]: There is a duplicated btn class assigned to this element. Maybe we should remove the duplicated.

File: setup.html [hunk 7]
```diff
@@ -5,18 +5,31 @@
...
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
```
Comment (jibm82) [inline setup.html]: Same here about the duplicated jquery reference.

File: setup.html [hunk 8]
```diff
@@ -5,18 +5,31 @@
...
+    <header class="masthead d-flex">
+        <div class="container text-center my-auto">
+            <img src="assets/images/logo_no-shadow.png" class="img-fluid mx-auto d-block" id="mixtape">
+
+            <section id="playlist-selection">
+                <div class="btn-group">
+                    <select class="form-control" id="playlist">
+                        <option value="">SELECT FROM YOUR PLAYLISTS</option>
+                    </select>
+                    <a class="btn btn" id="setup">LOAD</a>
```
Comment (jibm82) [inline setup.html]: Same here about the duplicated `btn` class.

File: assets/css/css/style.css [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+    cursor: pointer;
+    border: 1px solid #38f9d7;
+}
+
+#sub-btn:hover {
+    color: #ffffff;
+    background-color: #38f9d7;
+}
+
+.remainTime {
```
Comment (jibm82) [inline assets/css/css/style.css]: I couldn't locate this elements, are this rules relevant to our project?

File: setup.html (hunk 2 above)
Comment (edumoran) [reply setup.html]: checked

File: assets/css/css/reset.css (hunk 3 above)
Comment (edumoran) [reply assets/css/css/reset.css]: checked

File: index.html (hunk 4 above)
Comment (edumoran) [reply index.html]: checked

File: index.html (hunk 5 above)
Comment (edumoran) [reply index.html:17]: checked

File: setup.html (hunk 7 above)
Comment (edumoran) [reply setup.html]: checked

File: setup.html (hunk 8 above)
Comment (edumoran) [reply setup.html]: checked

File: assets/css/css/style.css (hunk 9 above)
Comment (edumoran) [reply assets/css/css/style.css]: This css was an old file I took to cut and paste over the new stylesheet, I removed it and only kept the one we are using.

**Review by jibm82 — APPROVED** [review]

**Comment by jibm82** [comment]
Thank you @edumoran, this one is ready to merge!
